apps/beeswax/src/beeswax/models.py

```diff
@@ -649,8 +649,7 @@ def to_dict(self):
       'description': self.description,
       'dialect': self.dialect,
       'interface': self.interface,
-      'external_id': self.external_id,
-      'last_modified': self.last_modified
```

Comment:
Why are we removing this field from `to_dict`?

Reply:
json conversion is failing for computes due to the `datetime` field. The field is only there for audit purposes and does not serve any functional, so this feels like the simplest solution.